src/three-components/SmoothControls.ts

```diff
@@ -494,6 +495,13 @@ export class SmoothControls extends EventDispatcher {
     const {maximumPolarAngle, maximumRadius, maximumFieldOfView} =
         this[$options];
 
+    const dTheta = this[$spherical].theta - this[$goalSpherical].theta;
+    if (Math.abs(dTheta) > Math.PI &&
+        !isFinite(this[$options].minimumAzimuthalAngle!) &&
+        !isFinite(this[$options].maximumAzimuthalAngle!)) {
+      this[$spherical].theta -= Math.sign(dTheta) * 2 * Math.PI;
```

Comment:
Substitute constant `TAU` above?

Reply:
I have a small vendetta against TAU, possibly just because I didn't go to MIT. I didn't even notice it was in the file. Personally I'd remove it, but I'll use it if you feel strongly.

Reply:
I don't feel strongly, but it was added at the request of another reviewer so I feel weird taking it out.

Use your discretion 👍